specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-10-01-preview/SourceControls.json

```diff
@@ -891,7 +891,7 @@
     },
     "Warning": {
       "description": "Warning response structure.",
-      "x-ms-external": true,
+      "x-ms-external": false,
```

Comment:
You can just remove this completely as the default behavior is the same as `false`.

Reply:
Thanks,  the file is updated as suggested. Please let me know if there is anything else to be changed.

Reply:
@jhendrixMSFT 
please let me know if anything else should be changed.